main.py

```diff
@@ -121,6 +136,36 @@ def build_details(details, report):
     }
 
 
+def post_comment(details):
+    def plural(count, word):
+        if count != 1:
+            return f"{count} {word}s"
+        return f"{count} {word}"
+
+    schema_doc_url = f"https://schema.data.gouv.fr/{details['schema_slug']}/latest.html"
+
+    comment = f"""\
+    Bonjour,
+
+    [Le fichier]({details["file_url"]}) que vous venez de publier ou mettre à jour comporte {plural(details["nb_errors"], "erreur")} sur un total de {plural(details["nb_rows"], "ligne")} par rapport au [schéma de référence]({schema_doc_url}).
+
+    Vous pouvez consulter le [dernier rapport de validation]({details["report_url"]}) pour vous aider à corriger les erreurs.
+    """
+
+    requests.post(
+        "https://www.data.gouv.fr/api/1/discussions/",
+        headers={
+            "X-API-KEY": os.environ["DATAGOUV_API_KEY"],
```

Comment:
@abulte Can you add the production key under Settings > Secrets for this repo?

Reply:
Done, you've been added to the alias validation@ too.